Lib/ldap/constants.py

```diff
@@ -0,0 +1,402 @@
+"""Definitions for constants exported by OpenLDAP
```

Comment:
Can you add a shebang, please?

Reply:
Should I? As a Linux packager, I have a bias against executable files in `/usr/lib`.
The module is intended to be run as `python Lib/ldap/constants.py > Modules/constants_generated.h`, and that doesn't need a shebang.